packs/feats/class/thaumaturge/intensify-investiture.json

```
@@ -29,7 +29,29 @@
             "remaster": false,
             "title": "Pathfinder Dark Archive"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "intensify-investiture",
+                "toggleable": true
+            },
+            {
+                "key": "AdjustModifier",
+                "mode": "override",
```

```suggestion
                "mode": "upgrade",
```
Does upgrade work? Do we actually need the toggle? While you could use the base DC there really isn't a situation where you would.

---

It might, I will check tomorrow. It needs to be a toggle because it's a limited use action.

---

Ah, then yeah just see if upgrade works. Also same for the alchemist PR.